iothub/device/samples/PnpDeviceSamples/PnpDeviceSamples/SimpleThermostat/Program.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Client;
+using Microsoft.Azure.Devices.Shared;
+
+namespace SimpleThermostat
+{
+    public class Program
+    {
+        private const string DeviceConnectionString = "device_connection_string_here";
+        private const string ModelId = "dtmi:com:example:simplethermostat;1";
+
+        private static DeviceClient s_deviceClient;
+
+        public static async Task Main(string[] _)
+        {
+            await RunSampleAsync().ConfigureAwait(false);
```

Comment:
This is a console app, so we don't need ConfigureAwait. Also I don't think we want to add this in samples based on what we heard in SDK review. @drwill-ms - Checking with you if we need to add it.